priv/templates/apple_captive_portal.html.eex

```diff
@@ -1,7 +1,7 @@
 <HTML>
   <HEAD>
     <TITLE>Captive Portal</TITLE>
-    <META http-equiv="refresh" content="0; url=<%= dns_name %>">
+    <META http-equiv="refresh" content="0; url=<%= dns_name %>" name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
here I would add a separate meta tag, instead of adding attributes to the existing one.